src/pages/ColoredManga/main.ts

```diff
@@ -0,0 +1,65 @@
+import { pageInterface } from '../pageInterface';
+import { getInter } from '../Isekaiscan/main';
+
+const clone = getInter();
```

Comment:
Title could be cleaned up a bit to make the matching with mal better.
Full Color
{Official TL}
(EN)